data/rulesets.js

```diff
@@ -717,16 +711,16 @@ exports.BattleFormats = {
 		},
 		onValidateTeam: function (team) {
 			let typeTable;
-			for (let i = 0; i < team.length; i++) {
-				let template = this.getTemplate(team[i].species);
-				if (!template.types) return [`Invalid pokemon ${team[i].name || team[i].species}`];
-				if (i === 0) {
+			for (const set of team) {
```

Comment:
Yes, `.entries()` is favorable, I just didn't realize you were still using `i` here... change it back?